revisions-digest.php

```
@@ -62,43 +62,152 @@ function widget( $no_idea, array $meta_box ) {
+	</form>
+	<?php
+}
 
-		echo '</div>';
+/**
+ * Render the list of existing subscriptions.
+ *
+ * @param array  $subscriptions The subscriptions array.
+ * @param string $nonce         The security nonce.
+ */
+function render_subscription_list( array $subscriptions, string $nonce ) : void {
```

PHPMD (2.15.0)

Avoid unused parameters such as '$nonce'. (undefined)

(UnusedFormalParameter)